### ISSUE TITLE
[FEATURE] Always output consistent wash dev ending

### ISSUE THREAD

**Issue by brooksmtownsend** [wasmCloud/wasmCloud#3460]
### Affected project(s)

- [ ] documentation
- [ ] examples
- [ ] wasmCloud host
- [X] wasmCloud CLI (wash)
- [ ] wasmCloud dashboard UI (washboard)
- [ ] capability providers
- [ ] provider bindgen
- [ ] control interface client
- [ ] other / not sure

### Is your feature request related to a problem? Please describe.

With wash dev, I can get frustrated when I forget the HTTP address to hit, or where the logs are located, after a few repeated builds

### Describe the solution you'd like

I think each devloop should output the same output, whether it was the first loop, a fast reload, or a dependency change. I'd propose that each dev loop has the following output:
```
🚧 Building project...
<build output>
✅ Successfully built project at [/Users/brooks/github.com/wasmcloud/wasmcloud/examples/rust/components/http-hello-world/build/http_hello_world_s.wasm]
ℹ️  Detected component dependencies: {"http-server"}
🔁 Deployed development manifest for application [dev-obyfxv-http-hello-world]
ℹ️  Logs available at /Users/brooks/.wash/dev/oBYfXv/wasmcloud.log
✨ HTTP Server: Access your application at http://127.0.0.1:8000
👀 Watching for file changes (press Ctrl+c to stop)...
```

So to summarize, this is:
1. The beginning of the devloop, where we build the project
2. Deployment information
3. Log location
4. HTTP URL and/or NATS messaging topic
5. Watching for changes...

With this solution, it's always easy to find out where to look for logs and where to hit the app
